plugins/Logs/js/logparse.js

```diff
@@ -0,0 +1,48 @@
+import fs from 'fs'
+import { readdirRecursive } from './utils.js'
+
+const siadir = SiaAPI.config.siad.datadir
+
+const fileSize = (logpath) =>
+	fs.statSync(logpath).size
```

Comment:
I usually stick short arrow function bodies on the same line if they're just a single expression. It's arguably less readable, but with a newline+indent it looks like python :P